cli/ballerina-cli/src/main/resources/cli-help/ballerina-profile.help

```diff
@@ -0,0 +1,27 @@
+NAME
+       ballerina-profile - Run Ballerina profiler on the source and generate flame graph
+
+SYNOPSIS
+       bal profile [OPTIONS] [<package>|<source-file>]
+
+
+DESCRIPTION
+       Compile the current package and run Ballerina profiler on it. This command generates an html file with name
+       `ProfilerOutput.html` in the target directory.
+
+       The generated `ProfilerOutput.html` file contains the flame graph that visualizes the distributed Ballerina
+       functions with execution details.
+
+OPTIONS
```

Comment:
Do we need --debug for profiler?

Reply:
yes. It is helpful to debug when there is a bug in the Profiler at runtime.